pkg/handlers/activate.go

```diff
@@ -0,0 +1,142 @@
+/****************************************************************************
```

Comment:
The description of this PR is all about adding a /decide API, isn't this adding a /activate api instead?

Reply:
nvm, just saw your comment in the PR. Maybe update the main PR description to minimize confusion?